lifetimes/estimation.py

```diff
@@ -4,7 +4,7 @@
 import numpy as np
 from numpy import log, exp, logaddexp, asarray, any as npany, c_ as vconcat,\
                   isinf, isnan, ones_like
-from pandas import DataFrame
+from pandas import DataFrame, ols
```

Comment:
yes, sorry it comes from a branch where i am trying to implement the pareto independent model in which i have to use it.